scripts/template_dir

```diff
@@ -182,17 +187,28 @@ do_cmake_package()
 
 # Read the input "y"
 # $1 is optional, but allows the caller to provide additional text.
-read_confirm () {
-  if [ "$confirm" != "y" ]; then
-      echo "Do you want to continue $1(y/n)? "
-      while read confirm; do
-          [ "$confirm" = "Y" -o "$confirm" = "y" -o "$confirm" = "n" \
-            -o "$confirm" = "N" ] && break
-          echo "Invalid input \"$confirm\", please input 'y' or 'n': "
-      done
-  else
-      echo
-  fi
+prompt () {
+	echo "Do you want to continue $1(y/n)? "
```

Comment:
The wording here is a bit confusing, if I saw:

"Do you want to continue registering Zephyr-sdk CMake module with path... (y/n)?"

I would have thought that hitting 'n' would have terminated the setup, Can we leave out the prefix and only standardize the "(y/n)?"

Reply:
+1 on this

Reply:
@alexeicolin any update on this feedback?

Reply:
Haven't got to this yet, I'll take a look.

Reply:
thanks, would like to get this in for 0.12.0, so not urgent, but would like to get this merged in the next week or 2 if possible.

Reply:
Removed the word 'continue' to prevent user confusion (pushed this as separate commit, because didn't really fit with any of the other commits).